prody/dynamics/plotting.py

```diff
@@ -2381,3 +2355,39 @@ def showTree_networkx(tree, node_size=20, node_color='red', node_shape='o',
         showFigure()
 
     return mpl.gca()
+
+
+def checkColors(colors, num, labels, allowNumbers=False):
+    """Check colors and process them if needed"""
+
+    from matplotlib.colors import is_color_like
+
+    colors_dict = {}
+
+    if is_color_like(colors) or colors is None or isinstance(colors, Number):
```

Comment:
Should `colors is None` be here? Wouldn't a list of None's fail the check down below?

Reply:
yes, it would fail so I added it down there too, because we do want to allow it